week-1/Homework/extra/1-currency-conversion.js

```diff
@@ -16,20 +18,24 @@ function convertToUSD() {}
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToBRL() {}
+function convertToBRL(pound) {
+  var total = pound * 5.7;
+  var final = total * 0.01;
+  return total + final;
+}
 
```

Comment:
As the instruction asks to add 1% to all conversions, that means both  convertToUSD, convertToBRL will get affected. 
And you can use in each one but that's the opposite of DRY principle 
You can create another function that does that
``` 
  var final = total * 0.01;
//OR use DRY
function addExtra(money){ 
return money*1.01;
}
```